silkworm/silkrpc/commands/ots_api.cpp

```diff
@@ -1043,7 +1043,7 @@ Task<BlockProviderResponse> ForwardBlockProvider::get() {
         }
     }
 
-    uint64_t next_block = next();
+    BlockNum next_block = next();
```

Comment:
The same change should be applied also in `ForwardBlockProvider::next` method both in `ots_api.hpp` and `ots_api.cpp`

Reply:
ok